ansible/library/generate_golden_config_db.py

```diff
@@ -101,10 +101,31 @@ def generate_smartswitch_golden_config_db(self):
         ori_config_db = json.loads(out)
         if "DEVICE_METADATA" not in ori_config_db or "localhost" not in ori_config_db["DEVICE_METADATA"]:
             return "{}"
-
         ori_config_db["DEVICE_METADATA"]["localhost"]["subtype"] = "SmartSwitch"
+
+        if "FEATURE" not in ori_config_db \
+                or "dhcp_server" not in ori_config_db["FEATURE"] \
+                or "dhcp_relay" not in ori_config_db["FEATURE"]:
+            return "{}"
+        ori_config_db["FEATURE"]["dhcp_server"]["state"] = "enabled"
+        ori_config_db["FEATURE"]["dhcp_relay"]["state"] = "enabled"
+
+        # Generate INTERFACE table for EthernetBPXX
+        if "PORT" not in ori_config_db or "INTERFACE" not in ori_config_db:
+            return "{}"
+        for i in range(8):
```

Comment:
What is the meaning of 8? Is it always 8?

Reply:
For cisco, it is alwasy 8.

Reply:
Seems it belongs to topo.yml. Suggest extend current topo infra.